scripts/deploy-to-gcs.sh

```diff
@@ -24,7 +24,7 @@ usage () {
 # GCS project
 GCS_PROJECT=es-repo
 # GCS bucket within project
-: "${GCS_BUCKET:=lightbend-helm-charts}"
+: "${GCS_BUCKET:=lightbend-console-charts}"
```

Comment:
Do we want to setup versioning on that bucket?  (I see that Lifecycle management is not currently enabled for this bucket.)

Reply:
Sure.

Reply:
@marcoderama can you do this, since you did it already? And please document the process in the Build and Release drive doc so we know how it was setup.

Reply:
Versioning is now setup.  I've set it to keep at most three versions of any object.

Still on me to update doco.